### PR TITLE
Get-DbaNetworkConfiguration: Output advanced properties

### PR DESCRIPTION
**Pull request by andreasjordan**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [x] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

Output advanced properties to now have all information accessable via SQL Server Configuration Manager under Properties of the SQL Server Network Configuration.

image

image

Will later update the Set-DbaNetworkConfiguration command to be able to change these settings.

But first let's have a look at the naming of variables and properties of the output - any suggestions for changing them? I have tried to name them like they appear on the GUI which is sometimes different than the name of the registry item property.

### CODE REVIEW

File: functions/Get-DbaNetworkConfiguration.ps1 [hunk 1]
```diff
@@ -135,9 +136,13 @@ function Get-DbaNetworkConfiguration {
             if ($regRoot) {
                 $regPath = "Registry::HKEY_LOCAL_MACHINE\$regRoot\MSSQLServer\SuperSocketNetLib"
                 try {
-                    $forceEncryption = switch ((Get-ItemProperty -Path $regPath -Name ForceEncryption).ForceEncryption) { 0 { $false } 1 { $true } }
+                    $acceptedNtlmSpns = (Get-ItemProperty -Path $regPath -Name AcceptedSPNs).AcceptedSPNs
```
Comment (wsmelton) [inline functions/Get-DbaNetworkConfiguration.ps1:139]: I'd be prone to match the name of the registry key. I think those that work with the advanced settings that much in SQL Server are not going to notice the difference, and likely use the registry key more than SSCM to manage it.

Comment (potatoqualitee) [reply functions/Get-DbaNetworkConfiguration.ps1:139]: yeah i do like the name of the registry keys, they are descriptive

**Comment by potatoqualitee** [comment]
happy to merge when it's been updated, thank you for this 👍🏼

File: functions/Get-DbaNetworkConfiguration.ps1 (hunk 1 above)
Comment (andreasjordan) [reply functions/Get-DbaNetworkConfiguration.ps1]: Ok, changed AcceptedNtlmSpns to AcceptedSPNs.

**Comment by andreasjordan** [comment]
Is updated.

**Comment by potatoqualitee** [comment]
👌🏼

**Pull request merged**